testsuite/layers/src/test/java/org/jboss/as/test/layers/LayersTestCase.java

```diff
@@ -26,6 +26,7 @@ public class LayersTestCase {
         // No patching modules in layers
         "org.jboss.as.patching",
         "org.jboss.as.patching.cli",
+        "org.wildfly.security.elytron-dynamic-ssl"
```

Comment:
If the intention is to leave the usage of this module when the server is trimmed including the Elytron layer, this change could be fine but we need to add a comment above it explaining the reason why this module is not provided when using layers. 

I have doubts we want that, it looks like the intention could be to also provide the capabilities of the dynamic SSL context even when we are using the Elytron layer. If that's the case, I guess you have to register the module with https://github.com/wildfly/wildfly-core/blob/d5456e9f7565513984b539ba494f9805863a36c6/controller/src/main/java/org/jboss/as/controller/ResourceDefinition.java#L127, which in turns should be invoked depending on the current Stability level

Reply:
> should be invoked depending on the current Stability level

or as a different way to see it, be invoked when the management resource associated with this Elytron Dynamic SSL is being registered, which I would assume is registered for community only stability level

Reply:
@yersan  yes, this module should be provided when using elytron layer and the community stability. I'll fix this, thank you!